scratch/benchmarks/sighandler.py

```diff
@@ -36,4 +36,6 @@ def handler(task: asyncio.Task) -> None:
         task.cancel()
 
     loop = asyncio.get_running_loop()
-    loop.add_signal_handler(signal.SIGINT, handler, asyncio.current_task())
+    task = asyncio.current_task()
+    assert task is not None
+    loop.add_signal_handler(signal.SIGINT, handler, task)
```

Comment:
Is this just to keep mypy happy or did it actually reveal a subtle bug?
(This is just a benchmark script I realise so it's not something we'd encounter often anyway but out of curiosity.)

Reply:
To keep mypy happy, although it could be None if this function was used incorrectly.